README.md

```diff
@@ -1,5 +1,7 @@
 This directory contains CFP design proposals for features impacting repos across the
-Cilium Github organization.
+Cilium Github organization. 
+
+All current CFPs are listed in the [Cilium Feature Proposals (CFPs) project]() found in this repository.
```

Comment:
Oh I see, I missed the PR description when I first read the PR :facepalm: 

> These statuses will be listed in the [Cilium Feature Proposals (CFP) GitHub Project](https://github.com/users/katiestruthers/projects/4/views/1) that will be added to this repository. Workflows are set to automatically set Provisional (when a new PR is opened), Implementable (when a PR is merged), and Withdrawn (when a PR is closed), but the other three statuses will have to be manually set. The SIGs will also have to be manually set.

How will we manage these? If this is automated somehow, great. If not, then I think this is creating more busywork that is likely to be neglected by the Cilium project unless you find someone specific who will volunteer to keep the CFP projects up to date. Almost every time we've tried to use projects or milestones etc. in the Cilium project, there's one person who pays attention to them for a few weeks, then they get stale and outdated.

Reply:
If this is something that SIG leads see value in and would be happy to help with, then I would encourage setting up something like that. On the other hand, if we don't have buy-in from anyone who would actively engage with this process, then I think it's the process that needs to change to fit the people, not the people changing to fit the process.

Reply:
I'll CC in specifically @marseel , @hemanthmalla (on behalf of @cilium/sig-scalability) and @squeed (on behalf of @cilium/sig-policy) in case they have opinions about how to track CFPs within specific SIGs and whether sharing a project to track these things would be helpful for them.